### DISCUSSION TITLE
How to run multiple instances of tgcf with different accounts in same machine ?

### DISCUSSION THREAD

**Discussion by ehs4nm** [aahnik/tgcf#248] [Q&A] [answered]
Is it possible to login in with multiple accounts?
how to run two different instance of tgcf in one machine?

i have tried to: 
1. use venv and it bothers a little.
2. install package with two py version (can't succeed!).
3. install the package with different name (can't succeed!).
4. run tgcf directly from the code (copy the code to a folder manually: i nedd some help here)

**Comment by aahnik** [comment, marked as answer]
Create two different folders. Use gnu screen. (Terminal multiplexer). Google search and self learn, how to install and use.

In the first folder create your configuration, and run tgcf using gnu screen. And then close the screen. Tgcf will keep running in background.

Now create another folder and do same.
